target/xilinx/flavor_vanilla/scripts/run.tcl

```diff
@@ -14,9 +14,15 @@ set_param general.maxThreads 8
 
 # Contraints files selection
 switch $::env(XILINX_BOARD) {
-  "genesys2" - "kc705" - "vc707" - "vcu128" - "zcu102" {
+  "vcu128" {
     import_files -fileset constrs_1 -norecurse constraints/$::env(XILINX_BOARD).xdc
-    import_files -fileset constrs_1 -norecurse constraints/$::env(XILINX_PROJECT).xdc
+    import_files -fileset constrs_1 -norecurse constraints/carfield_top_xilinx.xdc
```

Comment:
So to my understanding:

* `carfield.xdc` and `carfield_islands.tcl` are common constraints
* `carfield_top_xilinx.xdc` are specific of the vanilla implementation
* the `bd` does not have such specific constraints

Correct?

Reply:
I think bd does not need such constraints being that it uses Xilinx native IPs (in particular the DDR controller replacing the Hyperbus, not sure about JTAG and UART).

Reply:
Yes, certain the carfield top level constraints are defined in `carfield_xilinx_ip.xdc`,  but above the IP level Vivado handles it

Reply:
I will resolve this.